library/Ivoz/Cgr/Infrastructure/Cgrates/Service/BillingService.php

```diff
@@ -45,6 +45,36 @@ public function simulateCall(string $tenant, string $subject, string $destinatio
         );
     }
 
+    /**
+     * Simulate call and get billing details
+     *
+     * @param string $tenant
+     * @param string $subject
+     * @param int $durationSeconds
+     *
+     * @throws \DomainException
+     * @return SimulatedCall
+     */
+    public function simulateCallByRatingProfile(string $tenant, string $subject, string $destination, int $durationSeconds): SimulatedCall
```

Comment:
How were we executing this from klear?

Reply:
We did not execute this from Klear, this is the same function as simulateCall, but renamed, this is because you ask me to set a function called simulateCallByRatingProfile at the BillingService. 
![image](https://user-images.githubusercontent.com/101095813/215999375-9bb5314f-37f7-4f69-aacc-440fa500d9ed.png)